### ISSUE TITLE
Failure to parse wildcards in sysctl configuration file.

### ISSUE THREAD

**Issue by WavyEbuilder** [a13xp0p0v/kernel-hardening-checker#192]
Take a sysctl configuration file with the following contents:
```
net.ipv4.conf.*.accept_source_route = 0
```

Before:
```
rsandhu@graphite ~ $ cat /proc/sys/net/ipv4/conf/wlan0/accept_source_route
1
```
And after applying the sysctl file with `sysctl -p /etc/sysctl.d/00-test`:
```
rsandhu@graphite ~ $ cat /proc/sys/net/ipv4/conf/wlan0/accept_source_route
0
```

This seems to imply `sysctl(1)` supports wildcards. However, KHC fails with:
```
[-] ERROR: unexpected line in sysctl file: "net.ipv4.conf.*.accept_source_route = 0"
```

**Comment by a13xp0p0v** [comment]
Hello @WavyEbuilder, thanks for creating this report!

Fixed in https://github.com/a13xp0p0v/kernel-hardening-checker/commit/1a230d7fb3abc77333f61fbcc24a2127099c53ce.

Feel free to reopen it if the fix doesn't work for you.

By the way, I would recommend checking a file with the output of `sysctl -a` instead of `sysctl.conf` config, because it contains all sysctl parameters.

**Comment by WavyEbuilder** [comment]
Thanks! And noted
